pygfxd.py

```diff
@@ -853,6 +869,24 @@ def gfxd_macro_packets() -> int:
     """
     return lgfxd.gfxd_macro_packets()
 
+lgfxd.gfxd_foreach_pkt.argtypes = [CFUNCTYPE(c_int)]
+lgfxd.gfxd_foreach_pkt.restype = c_int
+def gfxd_foreach_pkt(fn: Callable[[], int]) -> int:
+    """
+        int fn(void))
```

Comment:
```suggestion
    int fn(void);
```
I guess this is just to notate the expected form of `fn`?